lib/src/sql/changefeed.rs

```diff
@@ -0,0 +1,69 @@
+use crate::sql::comment::shouldbespace;
+use crate::sql::error::IResult;
+use nom::bytes::complete::tag_no_case;
+use serde::{Deserialize, Serialize};
+use std::fmt::{self, Display, Formatter};
+use std::str;
+use std::time;
+
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize, Hash)]
+pub struct ChangeFeed {
+	pub expiry: time::Duration,
+}
+
+impl ChangeFeed {
+	pub fn enabled(d: time::Duration) -> Self {
+		ChangeFeed {
+			expiry: d,
+		}
+	}
```

Comment:
Is this now only used in the test?

Reply:
Correct! Probably we don't need it- I've just replaced it with `ChangeFeed {expiry: ...}`!